content/app/commands/goto-line-command.js

```diff
@@ -0,0 +1,76 @@
+(function (window) {
```

Comment:
Commands should throw events when they are completed / canceled. That would help command manager to know the state of it's commands and it would offer greater flexibility for further expansion of the manager.